recipe/bld.bat

```diff
@@ -0,0 +1,9 @@
+:: There is no `gdal-config` on Windows so we need to hardcode gdal's version.
+set GDAL_VERSION=3.1.4
```

Comment:
Maybe you can use something like this https://github.com/conda-forge/rasterio-feedstock/blob/master/recipe/bld.bat

Reply:
Ah, thanks. I copied it from fiona's feedstock ;)

BTW, I had expected that it would build against GDAL 3.1, since that's what is in the global pinnings (https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/2726e85da555e84e8ae7a2b4a33876495d285c68/recipe/conda_build_config.yaml#L461), but it actually built against GDAL 3.2 (latest available version). Did I misunderstand something about the pinnings?

Reply:
[The migration overrides the pin](https://github.com/conda-forge/pyogrio-feedstock/blob/master/.ci_support/migrations/gdal32.yaml). When the migration is done the pin is updated.

PS: we should updated fiona too!

Reply:
I see, thanks for the explanation!

Reply:
-> https://github.com/conda-forge/pyogrio-feedstock/pull/9